src/server/middleware/reactApplication/generateHTML.js

```diff
@@ -30,7 +30,7 @@ function scriptTags(jsFilePaths) {
 
 
 export default function generateHTML(args) {
-  const { reactAppString, initialState, nonce, helmet, codeSplitState } = args;
+  const { reactAppString, initialState, nonce, helmet, codeSplitState, styledComponents } = args;
```

Comment:
For someone not knowing the code, this could read like the `styledComponents` prop has something to do with components being rendered, while in reality it is just a simple string of CSS. Imho, it is not important to the `generateHTML` method where the generated styles are coming from and especially this naming is very confusing.